mantidproject/mantid
# MantidPython and Notebook remove and add Docs

**Description of work.**

Removed the MantidPython and Ipython Notebook from MacOS and added documentation (updated and moved to the docs from here: https://www.mantidproject.org/Using_IPython_Notebook) for launching them from the commandline.

The IPython Notebook package (clickable icon) still exists on Windows (although the MantidPython seems not to).

**To test:**

* Create a package build on Mac and note that the two packages have been removed.
* Build docs to check the new docs page.
* Have I definitely removed the correct files from `buildconfig/CMake/Packaging/osx/` ?


Fixes #28356 

*This does not require release notes* because **I'm not sure this is generally relevant**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
